py/float.go

```diff
@@ -394,6 +394,21 @@ func (a Float) M__ge__(other Object) (Object, error) {
 	return NotImplemented, nil
 }
 
+// Properties
+func init() {
+	FloatType.Dict["is_integer"] = MustNewMethod("is_integer", func(self Object) (Object, error) {
+		if a, ok := convertToFloat(self); ok {
+			f, _ := FloatAsFloat64(a)
+			if math.Floor(f) == f {
+				return True, nil
+			}
+			return False, nil
+		}
+		_, e := cantConvert(self, "float")
```

Comment:
```suggestion
		return cantConvert(self, "float")
```

Reply:
hth :)

Reply:
Thanks for helping me out.
I couldn't think so....